### ISSUE TITLE
[Usable – Make Code – Home page] Visually appearing ‘View all’ Button looks like normal text on ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3431]
**User Experience**:
User rely on keyboard will find difficulty to interact with the tooltip information if  tooltip is not accessible through keyboard as user won't be able to perform functionality over such control.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify the ‘View all’ button by using developer tool.

**Actual Result:**
Visually appearing ‘View all’ Button looks like normal text on ‘Home page’, due to this users will not be able to see the difference b/w text and button and will face difficulties while navigating on the page.
 
**Expected Result:**
Boundary lines should be define for ‘View all’ button under ‘Home page’, so that users can see the difference b/w text and button and will not face any difficulties while navigating on the page.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Reference Links:**
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
#URL:https://makecode.microbit.org/#

**Comment by Shalini3457** [comment]
![Uploading Usable_View All button looks as a normal text.jpg…]()

**Comment by Shalini3457** [comment]
![Usable_View All button looks as a normal text](https://user-images.githubusercontent.com/72871089/96090477-df34a100-0ee5-11eb-9c12-33716a210f99.jpg)
